mercury/templates/index.html

```diff
@@ -1,51 +1,69 @@
 {% load static %}
 <link rel="stylesheet" type="text/css" href="{% static 'mercury/style.css' %}">
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/js/bootstrap.min.js"></script>
 
 <!DOCTYPE html>
 <html>
 
   <head>
-      <meta charset="utf-8">
-      <title>Mercury - NYU Tandon Telemetry</title>
-      <link rel="icon" href="{% static 'mercury/favicon.png' %}">
-      <script>
-          function RedirectSimulator() {
-              window.location="/simulator/";
-          }
-          function RedirectDashboard() {
-              window.location="/dashboard/";
-          }
-          function RedirectCAN() {
-              window.location="/can/";
-          }
-          function RedirectStopwatch() {
-              window.location="/stopwatch/";
-          }
-          function newGrafanaWindow() {
-              window.open("https://nyu-mercury-grafana.herokuapp.com/d/BjN37GJZk/nyu-motorsports-telemetry-dashboard?orgId=1&refresh=5s");
-          }
-      </script>
+    <meta charset="utf-8">
+    <title>Mercury - NYU Tandon Telemetry</title>
+    <link rel="icon" href="{% static 'mercury/favicon.png' %}">
+    <script>
+      function RedirectSimulator() {
+        window.location="/simulator/";
+      }
+      function RedirectDashboard() {
+        window.location="/dashboard/";
+      }
+      function RedirectCAN() {
+        window.location="/can/";
+      }
+      function RedirectStopwatch() {
+        window.location="/stopwatch/";
+      }
+      function newGrafanaWindow() {
+        window.open("https://nyu-mercury-grafana.herokuapp.com/d/BjN37GJZk/nyu-motorsports-telemetry-dashboard?orgId=1&refresh=5s");
+      }
+    </script>
```

Comment:
I don't think any of this is required any more.

Reply:
Agreed, removing now.

Reply:
Oops, sorry. I resolved a conflict in the GitHub UI here and just removed it. You'll need to `git pull` on this simulatorcontent branch to pick up the change.

Reply:
just updated